exploration/function-composition-part-1.md

```diff
@@ -469,7 +469,10 @@ abstract "input" and "output" types.
 Returning to Example Y1, consider two possible models
 of the runtime behavior of function composition.
 
-**Model 1:**
+**Preservation Model (Model 1)**
```

Comment:
Why are the numbers still retained here (parenthetically)?

Reply:
Fixed